packages/grid/src/styled/StyledRow.spec.tsx

```diff
@@ -6,71 +6,140 @@
 
 describe('StyledRow', () => {
   it('renders default styling', () => {
     const { container } = render(<StyledRow />);
 
-    expect(container.firstChild).toHaveClass('row');
+    expect(container.firstChild).toHaveStyleRule('display', 'flex');
+    expect(container.firstChild).not.toHaveStyleRule('box-shadow'); /* debug = false */
   });
 
-  it('renders collapsed if provided', () => {
-    const { container } = render(<StyledRow isCollapsed={true} />);
+  it('renders debug styling if provided', () => {
+    const { container } = render(<StyledRow debug />);
 
-    expect(container.firstChild).toHaveClass('no-gutters');
+    expect(container.firstChild).toHaveStyleRule('box-shadow', expect.any(String));
   });
 
-  describe('Align Items', () => {
-    it('renders start alignment if provided', () => {
-      const { container } = render(<StyledRow alignItems="start" />);
-
-      expect(container.firstChild).toHaveClass('align-items-start');
-    });
-
-    it('renders center alignment if provided', () => {
-      const { container } = render(<StyledRow alignItems="center" />);
-
-      expect(container.firstChild).toHaveClass('align-items-center');
+  describe('Gutters', () => {
+    ARRAY_SPACE.forEach(size => {
+      if (size) {
+        it(`renders ${size} gutters`, () => {
+          const { container } = render(<StyledRow gutters={size} />);
+          const margin = math(`${DEFAULT_THEME.space[size]} / 2`);
+
+          expect(container.firstChild).toHaveStyleRule('margin-right', `-${margin}`);
+          expect(container.firstChild).toHaveStyleRule('margin-left', `-${margin}`);
+        });
+      } else {
+        it('collapses gutters', () => {
+          const { container } = render(<StyledRow gutters={false} />);
+
+          expect(container.firstChild).toHaveStyleRule('margin-right', '-0');
+          expect(container.firstChild).toHaveStyleRule('margin-left', '-0');
+        });
+      }
     });
+  });
 
-    it('renders end alignment if provided', () => {
-      const { container } = render(<StyledRow alignItems="end" />);
-
-      expect(container.firstChild).toHaveClass('align-items-end');
+  describe('Align Items', () => {
+    ARRAY_ALIGN_ITEMS.forEach(alignItems => {
+      it(`renders ${alignItems} flex alignment`, () => {
+        const { container } = render(<StyledRow alignItems={alignItems} />);
+
+        expect(container.firstChild).toHaveStyleRule(
+          'align-items',
+          expect.stringContaining(alignItems)
+        );
+      });
+
+      describe('Responsively', () => {
+        Object.keys(DEFAULT_THEME.breakpoints).forEach(breakpoint => {
+          const key = `alignItems${breakpoint[0].toUpperCase()}${breakpoint.substring(1)}`;
+
+          it(`renders ${key}=${alignItems} flex alignment`, () => {
+            const props = { [key]: alignItems };
+            const { container } = render(<StyledRow {...props} />);
+            const minWidth = (DEFAULT_THEME.breakpoints as any)[breakpoint];
```

Comment:
@austin94 I'm wondering if there's a better TS hack than the `as any` used here.

Reply:
There are probably some more specific semantics for this, but it makes sense to me 😄 Since this is a test file I don't think we need to change this. LGTM